.github/workflows/postrelease.yml

```diff
@@ -0,0 +1,12 @@
+name: Process new release
+
+on:
+  release:
+    types: [released] # don't run on draft releases or pre-releases
+
+jobs:
+  update-tap:
+    permissions:
+      contents: write
+    secrets: inherit
+    uses: restatedev/homebrew-tap/.github/workflows/update.yml@main
```

Comment:
Are you sure that this works and we are not trying to run the `update.yml` workflow in the context of the `restate` repo?

Reply:
it will run it in this repo, so update.yml needs to checkout the right repo - ive sorted that

Reply:
Great, thanks.